src/AutoRest.CSharp.V3/CodeGen/ClientWriter.cs

```diff
@@ -462,21 +529,14 @@ ref valueVariable
                         writer.Line($"var headers = new {headersModelType}(message.Response);");
                     }
 
-                    switch (responseBody)
+                    var responseCallText = responseBody switch
```

Comment:
this is a wrong change, you care preformating FormattableString into a normal string and loosing type information embedded in the FormattableString. Logic we use to add usings won't work correctly. Undo.

Reply:
Fixed.